src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyRepositoryTests.cs

```diff
@@ -0,0 +1,310 @@
+            {
+                await contentDbContext.AddAsync(publication);
+                await contentDbContext.AddAsync(release);
+                await contentDbContext.AddAsync(userReleaseRole);
+                await contentDbContext.AddAsync(methodology);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var methodologyRepository = new MethodologyRepository(contentDbContext);
+                var methodologies = await methodologyRepository.GetMethodologiesForUser(userId);
+
+                Assert.Empty(methodologies);
+            }
+        }
+        
+        [Fact]
+        public async Task GetMethodologiesForUser_Multiple()
+        {
+            var userId = Guid.NewGuid();
+            var publication1 = new Publication
+            {
+                Id = Guid.NewGuid(),
+                Title = "Test pub 1"
+            };
+            var release1 = new Release
+            {
+                Id = Guid.NewGuid(),
+                PublicationId = publication1.Id
+            };
+            var userReleaseRole1 = new UserReleaseRole
+            {
+                UserId = userId,
+                Release = release1,
+                Role = ReleaseRole.Lead,
+            };
+            var methodology1 = new Methodology
+            {
+                Id = Guid.NewGuid(),
+                Title = "Test methodology 1",
+                Publications = new List<Publication> {publication1}
+            };
+
+            var publication2 = new Publication
+            {
+                Id = Guid.NewGuid(),
+                Title = "Test pub 2"
+            };
+            var release2 = new Release
+            {
+                Id = Guid.NewGuid(),
+                PublicationId = publication2.Id
+            };
+            var userReleaseRole2 = new UserReleaseRole
+            {
+                UserId = userId,
+                Release = release2,
+                Role = ReleaseRole.Contributor,
+            };
+            var methodology2 = new Methodology
+            {
+                Id = Guid.NewGuid(),
+                Title = "Test methodology 2",
+                Publications = new List<Publication> {publication2}
+            };
+            
+            var publication3 = new Publication
+            {
+                Id = Guid.NewGuid(),
+                Title = "Test pub 3"
+            };
+            var release3 = new Release
+            {
+                Id = Guid.NewGuid(),
+                PublicationId = publication3.Id
+            };
+            var userReleaseRole3 = new UserReleaseRole
+            {
+                UserId = Guid.NewGuid(),
+                Release = release3,
+                Role = ReleaseRole.Lead,
+            };
+            var methodology3 = new Methodology
```

Comment:
Don't have to do it, but I personally like to add comments on these excluded/negative cases describing why they won't be included in the final result. Makes it easier for the reader to quickly see what's going on :slightly_smiling_face: